File: infrastructure/spring-based-app/work-log-web/src/app/model/work-log.ts
import {Worker} from './worker';

export class WorkLog {

    workLogId: number;
    startDate: Date;
    timeSpentInSeconds: number;
    associatedWorker: Worker;
    description: string;
    constructor(workLogId: number, startDate: Date, timeSpentInSeconds: number, associatedWorker: Worker, description: string) {
        this.workLogId = workLogId;
        this.startDate = startDate;
        this.timeSpentInSeconds = timeSpentInSeconds;
        this.associatedWorker = associatedWorker;
        this.description = description;
    }


}

File: domain/src/main/java/pl/splaw/domain/model/WorkLog.java
package pl.splaw.domain.model;

import java.math.BigInteger;
import java.time.LocalDate;
import lombok.AllArgsConstructor;
import lombok.Getter;


/**
 *
 * @author Bartek <https://github.com/splaw88>
 */
@Getter
@AllArgsConstructor
public class WorkLog {

    private final Long workLogId;
    private final LocalDate startDate;
    private final BigInteger timeSpentInSeconds;
    private final Worker associatedWorker;
    private final String description;

}
